pkg/cluster/module/user.go

```diff
@@ -48,6 +49,7 @@ var (
 type UserModuleConfig struct {
 	Action string // add, del or modify user
 	Name   string // username
+	Group  string // group name
```

Comment:
There is no command line flag to assign a value to `Group`.

Reply:
There is no need to add them to command line because the EnvInit only read user from topology file.